api/v1alpha2/virtualmachine_types.go

```diff
@@ -380,6 +380,13 @@ type VirtualMachineSpec struct {
 	// behavior. In other words, please be careful when choosing to upgrade a
 	// VM to a newer hardware version.
 	MinHardwareVersion int32 `json:"minHardwareVersion,omitempty"`
+
+	// BiosUUID describes the desired BIOS UUID for a VM.
+	// If omitted, this field defaults to a random UUID.
+	// When the bootstrap provider is Cloud-Init, this value is
+	// used as the Cloud-Init instance ID.
+	// +optional
+	BiosUUID string `json:"biosUUID,omitempty"`
```

Comment:
Can you set this field in the big https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha1/virtualmachine_conversion_test.go#L61 conversion test to make sure that its preserved on conversions?

Reply:
Thanks for pointing that out, done.